src/js/media-chrome-range.js

```diff
@@ -151,27 +159,6 @@ template.innerHTML = `
       height: var(--media-range-track-height, 4px);
     }
 
-    ${/*
-     * set input to focus-visible, unless host-context is available (in chrome)
-     * in which case we can have the focus ring be on the host itself
-     */''}
-    :host-context([media-keyboard-control]):host(:focus),
-    :host-context([media-keyboard-control]):host(:focus-within) {
-      box-shadow: inset 0 0 0 2px rgba(27, 127, 204, 0.9);
-    }
-    :host-context([media-keyboard-control]) input[type=range]:focus-visible {
-      box-shadow: none;
-    }
```

Comment:
I can be wrong but I believe this couldn't work if this control was not in the media-controller element.
more high level `:host-context` is probably not a good solution for loose coupled components which is one the goals of media-chrome.

I know the solution requires some extra JS but since `media-keyboard-control` is gotten from event listeners too seems this is a good enough solution listening for focusin / focusout events and checking .matches(':focus-visible')

Reply:
I wonder if instead we want to move some of the media-container stuff like this into a base media-chrome element that everything extends (or a mixin or something).

However, yeah, your code does seem like it should work.